src/components/text/_text.scss

```
@@ -117,9 +117,8 @@
     padding: fontSizeToRemOrEm(($baseLineHeightMultiplier * 2), $sizingMethod);
   }
 
-  code {
-    // Setting the font size in rem's ensure it doesn't compound
-    // the 90% set by euiCodeBlock
+  code:not(.euiCode):not(.euiCodeBlock__code) {
+    // Excluding EuiCode and EuiCodeBlock selectors to to compound the reduced font-size
```

@cchaos should this be to *not* compound the reduced font-size?

```suggestion
    // Excluding EuiCode and EuiCodeBlock selectors to not compound the reduced font-size
```

---

Yerp 🤦‍♀️